lib/variable/variable_instantiate_bin_elements.cpp

```diff
@@ -26,8 +26,8 @@ template <class T> class BinVariableMakerVariable : public BinVariableMaker<T> {
                           const bool variances) const override {
     // Buffer contains only variable, which is created with new dtype, no
     // information to copy from parent.
-    return make_bins(copy(indices), dim,
-                     variableFactory().create(type, dims, unit, variances));
+    return make_bins_no_validate(
+        indices, dim, variableFactory().create(type, dims, unit, variances));
```

Comment:
Can this now lead to sharing indices? Or is this function only called with a fresh set of indices that the user has no handle of?

Reply:
As far as I can tell it is always fresh, but please double check! The intention is that this does *not* lead to sharing.

Reply:
Looks like it is only called in one place and that passes in freshly created indices.